### PR TITLE
Fixes https://github.com/git-for-windows/git/issues/2873

### PR DESCRIPTION
**Pull request by mhoyer**

### CODE REVIEW

File: openssh/PKGBUILD [hunk 1]
```diff
@@ -14,7 +14,8 @@ source=("https://cloudflare.cdn.openbsd.org/pub/OpenBSD/OpenSSH/portable/${pkgna
         0001-Forward-port-MSys2-patches.patch
         0002-openssh-work-around-Cygwin-declaring-setkey.patch
         0003-openssh-skip-privilege-separation-tests.patch
-        0004-Allow-scp-to-copy-files-that-start-with-a-Windows-dr.patch)
+        0004-Allow-scp-to-copy-files-that-start-with-a-Windows-dr.patch
+        d9e727dcc04a52caaac87543ea1d230e9e6b5604.patch)
```
Comment (dscho) [inline openssh/PKGBUILD:18]: I think that file is missing.

Also, may I ask you to make the commit message more stand-alone? Something like: "openssh: fix copy-ssh-id" with a brief description of the symptoms fixed by the commit.

Comment (mhoyer) [reply openssh/PKGBUILD:18]: 🤦 Man, how could I miss that one. Does it look better now?

**Review by dscho — APPROVED** [review]

File: openssh/PKGBUILD (hunk 1 above)
Comment (dscho) [reply openssh/PKGBUILD:18]: Very nice!

**Comment by dscho** [comment]
The Pacman package is built here: https://dev.azure.com/Git-for-Windows/git/_build/results?buildId=65084&view=results

**Pull request merged**